src/welcome/ui/components/SeedConfirmation.tsx

```diff
@@ -1,65 +1,109 @@
-import { MouseEventHandler, useEffect, useState } from 'react'
+import { MouseEventHandler, useState, useEffect } from 'react'
+import { pickMultipleRandomly } from '../../helpers/random'
 // Material UI
-import Alert from '@mui/material/Alert'
 import Box from '@mui/material/Box'
+import Grid from '@mui/material/Grid'
 import Button from '@mui/material/Button'
 import Divider from '@mui/material/Divider'
-import TextField from '@mui/material/TextField'
 import Typography from '@mui/material/Typography'
 // Components
 import NoShareWarning from './NoShareWarning'
+import WordInput from './WordInput'
 
-export default function SeedConfirmation(props: {
+const WORDS_TO_CONFIRM = 3
```

Comment:
If we want more words to be confirmed, the UI looks nice with 6. If we want something different, like 4, we may want to tweak it a little bit.